Sources/Commands/SwiftBuildTool.swift

```diff
@@ -105,7 +105,8 @@ public struct SwiftBuildTool: SwiftCommand {
         }
 
         if options.printManifestGraphviz {
-            let buildOperation = try swiftTool.createBuildOperation()
+            // FIXME: Doesn't seem ideal that we need an explicit build operation, but this concretely uses the `LLBuildManifest`.
```

Comment:
I don't see a good way to solve this right now, so I am leaving this for now. In the future, this could maybe be an extra command vended by a potential build system plugin and would that way only be available for a specific build system that integrates with LLBuild.